forest/Cargo.toml

```diff
@@ -12,6 +12,7 @@ async-std              = { version = "1.9", features = ["attributes", "tokio1"]
 base64                 = "0.13"
 byte-unit              = "4.0"
 cid                    = { version = "0.8", default-features = false, features = ["std"] }
+console-subscriber     = "*"
```

Comment:
That's a bit optimistic in terms of backwards compatibility, maybe it would be better to pin it to the current minor, i.e. `console-subscriber = "0.1.7"`?

Reply:
Yup.